mage/ginkgo/lib.go

```diff
@@ -417,7 +417,7 @@ func CreateDockerRegistrySecret(name string, namespace string) {
 
 func (ns NsWrapper) HelmInstall(chartPath string, overrides ...string) {
 	overridesMap := map[string]string{}
-	for i := 0; i < len(overrides) - 1; i = i+2 {
+	for i := 0; i < len(overrides)-1; i = i + 2 {
```

Comment:
:confused:   I guess this is go standards and not your preferences, but why no consistency, Golang??

Reply:
oh yeah, I have gofmt running on save in spacemacs and it constantly finds things like this to change. I'm actually surprised it removed spaces around the `-`